remote/bitbucketserver/client.go

```diff
@@ -0,0 +1,61 @@
+package bitbucketserver
+
+import (
+	"net/http"
+	"crypto/tls"
+	log "github.com/Sirupsen/logrus"
+	"io/ioutil"
+	"encoding/pem"
+	"crypto/x509"
+	"github.com/mrjones/oauth"
+)
+
+
+func NewClient(ConsumerKey string, URL string) *oauth.Consumer{
+	privateKeyFileContents, err := ioutil.ReadFile("/private_key.pem")
+	log.Info("Tried to read the key")
```

Comment:
no worries!  In this case I'd recommend making the `private_key.pem` path configurable or possibly including as a bas64 string so that it doesn't need to get mounted into the container. How long is the pem file?